src/resolve.c

```diff
@@ -2659,6 +2659,19 @@ resolve_partial_json_data_nodeid(const char *nodeid, const char *llist_value, st
     goto parse_predicates;
 
     while (1) {
+        /* find the correct parent schema node for rpc */
+        sparent = (start && start->parent) ? start->parent->schema : NULL;
```

Comment:
sparent : This pointer is not declared

Reply:
oh... I forgot it when I copy the code. It should be fixed now.

Reply:
I just checked your issue again, it seems that is's not same with mine, and may not be fixed by this request. Sorry for that...